docs/develop/convert-javascript-to-typescript.md

```diff
@@ -14,95 +14,79 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
 
 - Excel 2016 or later
 
 ## Create the add-in project
 
-1. Open Visual Studio and on the Visual Studio menu bar, choose  **File** > **New** > **Project**.
+1. Choose **Create a new project**.
 
-2. In the list of project types under **Visual C#** or **Visual Basic**, expand  **Office/SharePoint**, choose **Add-ins**, and then choose **Excel Web Add-in** as the project type. 
+2. Using the search box, enter **add-in**. Choose **Excel Web Add-in**, then select **Next**.
 
-3. Name the project, and then choose **OK**.
+3. Name your project and select **Create**.
 
-4. In the **Create Office Add-in** dialog window, choose **Add new functionalities to Excel**, and then choose **Finish** to create the project.
+3. In the **Create Office Add-in** dialog window, choose **Add new functionalities to Excel**, and then choose **Finish** to create the project.
 
-5. Visual Studio creates a solution and its two projects appear in **Solution Explorer**. The **Home.html** file opens in Visual Studio.
+4. Visual Studio creates a solution and its two projects appear in **Solution Explorer**. The **Home.html** file opens in Visual Studio.
 
 ## Convert the add-in project to TypeScript
 
-1. In **Solution Explorer**, rename the **Home.js** file to **Home.ts**.
+1. Find the **Home.js** file and rename it to **Home.ts**.
 
-    > [!NOTE]
-    > In your TypeScript project, you can have a mix of TypeScript and JavaScript files and your project will compile. This is because TypeScript is a typed superset of JavaScript that compiles JavaScript. 
-
-2. Select **Yes** when prompted to confirm that you want to change file name extension.
-
-3. Create a new file named **Office.d.ts** in the root of the web application project.
-
-4. In a web browser, open the [type definitions file for Office.js](https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js/index.d.ts). Copy the contents of this file to your clipboard.
+2. From the **Tools** tab, choose **NuGet Package Manager** and then select **Manage NuGet Packages for Solution...**.
 
-5. In Visual Studio, open the **Office.d.ts** file, paste the contents of your clipboard into this file, and save the file.
+3. With the **Browse** tab selected, enter **office-js.TypeScript.DefinitelyTyped** into the search box. Install or update this package if it is already installed. This will add the TypeScript type definitions for the Office-js library to your project.
```

Comment:
```suggestion
3. With the **Browse** tab selected, enter **office-js.TypeScript.DefinitelyTyped** into the search box. Install or update this package if it is already installed. This will add the TypeScript type definitions for the Office.js library to your project.
```